src/Nest/XPack/MachineLearning/Job/Detectors/MetricDetector.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Linq.Expressions;
+
+namespace Nest
+{
+	public class MetricFunctions
```

Comment:
Can this be `private` and live in a separate file ?

Reply:
Related to above.